### PR TITLE
[CDAP-15386] Fix issue NPE when Pub/Sub event does not have attributes.

### PR DESCRIPTION
**Pull request by nitinmotgi**
Reopening https://github.com/data-integrations/google-cloud/pull/101 against release branch

### CODE REVIEW

**Comment by nitinmotgi** [comment]
@anwar6953 could you and provide lgtm if all good. This should go in 6.0

**Review by sreevatsanraman — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -58,8 +58,8 @@
     <jee.version>7</jee.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <avro.version>1.7.7</avro.version>
-    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
-    <cdap.plugin.version>2.2.0-SNAPSHOT</cdap.plugin.version>
+    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
```
Comment (anwar6953) [inline pom.xml:61]: why change this?
This should remain 6.0.0-SNAPSHOT, since this is the branch used to build cdap 6.0.

Comment (anwar6953) [reply pom.xml:61]: Ah, I think its because you started your PR branch from develop.

Comment (anwar6953) [reply pom.xml:61]: Undoing the change in https://github.com/data-integrations/google-cloud/pull/104.

**Pull request merged**